Tasks/DockerV0/containerbuild.ts

```diff
@@ -63,5 +63,11 @@ export function run(connection: ContainerConnection): any {
         context = tl.getPathInput("context");
     }
     command.arg(context);
+
+    command.on("stdout", data => {
+        let taskOutputPath = utils.writeTaskOutput("build", data);
```

Comment:
This should happen at the end (`stdout` event can happen in chunks).
On `stdout` event, append to a local variable and write to file and set the variable after command execution completes.

Reply:
`Suggestion`:
```
return connection.execCommand(command).then(() => {
  // write to file and output variable.
});```